test/unit/config/article_scrape_patterns_test.rb

```diff
@@ -55,4 +55,40 @@ def test_scrape_methods_presets_are_valid
       end
     end
   end
+
+  def test_all_preset_xpaths_are_valid
+    supported_domains = YAML.load_file('config/article_scrape_patterns.yml')['domains'].keys
+    noko_html = Nokogiri::HTML(raw_data_fixture(supported_domains.first))
+
+    @scrape_patterns['presets'].each_pair do |data_type, presets|
+      presets.each_pair do |preset_type, spec|
+        next unless spec['method'] == 'xpath'
+
+        begin
+          noko_html.xpath("(#{spec['pattern']})[1]")
+        rescue => e
```

Comment:
It varies depending on how you fail, so its better to catch everything.
It shouldn't raise otherwise.